px-vis-register-item.html

```diff
@@ -201,6 +201,17 @@
               break;
 
             case 'ordinal':
+              var xTimestamp = this.data.value[xKey];
+        
+              xVal = this._formattedX;
+              // checks if registerFormatCallback is avaliable, if so executes it expecting a return value for xVal
+              // allowing the user to set their own functionality.
+              if(this.completeSeriesConfig && this.registerFormatCallback) {
```

Comment:
If you're adding new props, they must be declared as properties [here](https://github.com/predixdesignsystem/px-vis/blob/master/px-vis-register-item.html#L136) and on any other component your prop passes through (eg. `px-vis-register`)

Reply:
added

Reply:
You're still missing the prop on `px-vis-register`.